core/system/src/core_armv7m/virq.c

```diff
@@ -519,17 +540,17 @@ void virq_init(uint32_t const * const user_vtor)
      * levels. */
     assert(__UVISOR_NVIC_MIN_PRIORITY < UVISOR_VIRQ_MAX_PRIORITY);
 
-    /* Set the priority of each exception. SVC is lower priority than
-     * MemManage, BusFault, UsageFault, and SecureFault_IRQn, so that we can
-     * recover from security violations more simply. */
+    /* Set the priority of each exception. SVC is set to higher priority than
```

Comment:
Please change the priority of exceptions in its own commit, so we can test that changing the priority doesn't cause us to fail tests, independently of other changes you've made.

Reply:
Made it a separate commit in this PR.

Reply:
OK

Thanks

Reply:
Changed the design to reset active bits, so reverted this change.